ssl/s3_lib.c

```diff
@@ -2823,7 +2823,7 @@ static SSL_CIPHER ssl3_ciphers[] = {
      0, 0,
      SSL_NOT_DEFAULT | SSL_MEDIUM,
```

Comment:
MEDIUM or worse for these?

Reply:
Not sure - nothing is at the low level currently. Note that this changes the sorting order when you modify this value - so I didnt want to mess with that here..